pkgs/by-name/jj/jjui/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+pkgs.buildGoModule rec {
+  pname = "jjui";
+  version = "b1f467fcef89a6f9e57a5be2108bb0ef34f1442d";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "idursun";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Ig4Amc9zc6rfwh4CyBhhk05/5Y8JpBT8OflxHXtTuOs=";
+  };
+
+  vendorHash = "sha256-kg5b3tzwyAhn00GwdUDf4OdYZvCJZHgkgpzHFWy5SxI=";
+
+  doCheck = false; # NOTE: Tests are disabled due to a bug in the tests upstream. Uncomment when the bug is fixed.
```

Comment:
A link to the upstream issue would be nice

Reply:
Good idea. Added.

Reply:
The failing tests, caused by unstable output of the tool, have been removed entirely. Therefore, disabling the tests in no longer necessary.